flutter-studio/testSrc/com/android/tools/idea/tests/gui/framework/fixture/IdeaFrameFixture.java

```diff
@@ -492,15 +492,15 @@ private ActionButtonFixture findActionButtonByText(@NotNull String text) {
     return ActionButtonFixture.findByText(text, robot(), target());
   }
 
-  @NotNull
-  public AndroidToolWindowFixture getAndroidToolWindow() {
-    return new AndroidToolWindowFixture(getProject(), robot());
-  }
-
-  @NotNull
-  public CapturesToolWindowFixture getCapturesToolWindow() {
-    return new CapturesToolWindowFixture(getProject(), robot());
-  }
+  //@NotNull
```

Comment:
Are these no longer needed? We should delete, or add a comment about why they're commented out for future readers.

Reply:
OK, I can delete them.